src/Oro/Bridge/ContactUs/Controller/ContactRequestController.php

```diff
@@ -34,7 +34,7 @@ public function createAction(Request $request)
         $contactRequest = new ContactRequest();
 
         $form = $this->createForm(
-            ContactRequestType::class,
+            $this->get('oro_contact_us_bridge.contact_request_type'),
```

Comment:
@mrfroasty, any reason to have service instead of basic approach?
If you need to customize that form, please read https://symfony.com/doc/current/form/create_form_type_extension.html topic.

Reply:
@manowark 
Sorry for the late reply. At the time I wanted to make a field "organizationName" to have required set to true, but via controller its the same core form that is being loaded. Probably you are right that there are other ways to achieve this than overriding this \Oro\Bridge\ContactUs\Form\Type\ContactRequestType.